src/preamble.js

```diff
@@ -1918,7 +1918,7 @@ addOnPreRun(function() {
       Runtime.loadDynamicLibrary(lib);
     });
   }
-  asm['runPostSets']();
+  asm['runPostSets'] && asm['runPostSets']();
```

Comment:
Yes, test_cubescript: https://github.com/kripken/emscripten/pull/4560/commits/95b375bf69302ffc9037372d8a40ce8c06d7c85d

Not completely confident about what `runPostSets` is supposed to do. My guess is that it runs a fixup-type thing that's inserted into the module after linking is over, but in this test case we're not actually linking anything, so there's no `runPostSets` to call. That seemed conceptually ok to me, so just don't call anything here.